corehq/toggles/__init__.py

```diff
@@ -1019,12 +1019,14 @@ def _ensure_valid_randomness(randomness):
 
 GEOCODER_USER_PROXIMITY = StaticToggle(
     "geocoder_user_proximity",
-    "USH: Set the geocoder widget's proximity based on the user's location.",
+    "USH: Adjust geo coder result to be more relevant to user and project.",
```

Comment:
```suggestion
    "USH: Adjust geocoder result to be more relevant to user and project.",
```

Reply:
Another nitpick: maybe "Adjust geocoder in case search..." to be clearer immediately that this doesn't affect geocoders in forms?